src/app/service/game-control.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, filter, map, scan } from 'rxjs';
+
+export type move = {
+  field: number;
+  player: string;
+};
+@Injectable({
+  providedIn: 'root',
+})
+export class GameControlService {
+  private gameBoard: BehaviorSubject<string[]> = new BehaviorSubject<string[]>(
```

Comment:
Can't it be just a number?

Reply:
I used behaviorSubject because, with it, I can easily reload the board in view